src/main/java/com/actiontech/dble/server/parser/RwSplitServerParse.java

```diff
@@ -128,6 +128,12 @@ public static int parse(String stmt) {
         return rt;
     }
 
+
+    public static boolean isMultiStatement(String sql) {
+        final int index = sql.indexOf(';');
```

Comment:
whether to consider value of insert has ';'

Reply:
No, I will fix it right away.